README.md

```diff
@@ -1,104 +1,127 @@
-### Specifying the Language
+#### Specifying the Language
 
-The language is indicated by a three-letter
-[ISO 639-2](https://en.wikipedia.org/wiki/List_of_ISO_639-2_codes) or
-[ISO 639-3](https://en.wikipedia.org/wiki/List_of_ISO_639-3_codes)
-language code, e.g., `fra` for French.
-For which languages can be scraped,
+The language is indicated by a three-letter [ISO
+639-2](https://en.wikipedia.org/wiki/List_of_ISO_639-2_codes) or [ISO
+639-3](https://en.wikipedia.org/wiki/List_of_ISO_639-3_codes) language code,
+e.g., `fra` for French. For which languages can be scraped,
 [here](https://en.wiktionary.org/wiki/Category:Terms_with_IPA_pronunciation_by_language)
 is the complete list of languages on Wiktionary that have pronunciation entries.
 
-### Output
+#### Output
 
-The scraped data is organized with each <word, pronunciation> pair on its
-own line, where the word and pronunciation are separated by a tab.
-Note that the pronunciation is in
-[International Phonetic Alphabet (IPA)](https://en.wikipedia.org/wiki/International_Phonetic_Alphabet),
-segmented by spaces that correctly handle the combining and modifier diacritics
-for modeling purposes,
-e.g., we have `kʰ æ t` with the aspirated k instead of `k ʰ æ t`.
+The scraped data is organized with each \<word, pronunciation\> pair on its own
+line, where the word and pronunciation are separated by a tab. Note that the
+pronunciation is in [International Phonetic Alphabet
+(IPA)](https://en.wikipedia.org/wiki/International_Phonetic_Alphabet), segmented
+by spaces that correctly handle the combining and modifier diacritics for
+modeling purposes, e.g., we have `kʰ æ t` with the aspirated k instead of
+`k ʰ æ t`.
 
 For illustration, here is a snippet of French data scraped by WikiPron:
 
-```tsv
-accrémentitielle	a k ʁ e m ɑ̃ t i t j ɛ l
-accrescent	a k ʁ ɛ s ɑ̃
-accrétion	a k ʁ e s j ɔ̃
-accrétions	a k ʁ e s j ɔ̃
+``` {.tsv}
+accrémentitielle    a k ʁ e m ɑ̃ t i t j ɛ l
+accrescent  a k ʁ ɛ s ɑ̃
+accrétion   a k ʁ e s j ɔ̃
+accrétions  a k ʁ e s j ɔ̃
 ```
 
-By default, the scraped data appears in the terminal.
-To save the data in a TSV file,
-please redirect the standard output to a filename of your choice:
+By default, the scraped data appears in the terminal. To save the data in a TSV
+file, please redirect the standard output to a filename of your choice:
 
-```bash
+``` {.bash}
 wikipron fra > fra.tsv
 ```
 
-### Advanced Options
+#### Advanced Options
 
-The `wikipron` terminal command has an array of options to configure
-your scraping run.
-For a full list of the options, please run `wikipron -h`.
+The `wikipron` terminal command has an array of options to configure your
+scraping run. For a full list of the options, please run `wikipron -h`.
 
-### Python API
+Python API
+----------
 
-The underlying module can also be used from Python.
-A standard workflow looks like:
+The underlying module can also be used from Python. A standard workflow looks
+like:
 
-```python
+``` {.python}
```

Comment:
It looks like this line has to stay as-is for syntax highlighting to work. I was looking [here](https://github.com/kylebgorman/wikipron/tree/readme) to see how the README would look after this PR, where the Python code highlighting didn't show up.

Reply:
I just went through and manually reverted all of those. I'm using Pandoc to reflow.